.gitmodules

```diff
@@ -0,0 +1,3 @@
+[submodule "test-repo"]
```

Comment:
I'm not sure if we need a submodule, is it necessary for running the tests?

Reply:
it's only necessary to avoid the data being constantly updated whenever we run the tests on `localhost`. 💭